data/ships.txt

```diff
@@ -203,7 +203,8 @@ ship "Bactrian"
 		"Ramscoop"
 		
 		"X4700 Ion Thruster"
-		"X5200 Ion Steering"
+		"X1700 Ion Thruster"
+		"X4200 Ion Steering"
```

Comment:
I must now point out that no (stock, not sure about variants) human ship has stacked engines.

And even if you do intend to stack them, is steering not valued more than thrust on massive vessels like bactrians?